docs/pages/versions/unversioned/guides/contributing-to-web.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Contributing to Web
+---
+
+There are a couple different things you could work on in regards to Expo for web. Here we'll cover how you can setup your environment to easily contribute.
+
+## 💎 Features & Tests
+
+To work on modules like `expo-camera` or even community libraries, I like to work in the expo monorepo, this is because it's a yarn workspace which links the packages really well.
+
+- Clone the repo `git clone git@github.com:expo/expo.git`
+- [Set up the project](https://github.com/expo/expo/#set-up).
+- I also tend to run `git submodule init && git submodule update && git submodule foreach --recursive git checkout . && git lfs pull` just to be safe! 😄
+
+Now to develop Unimodules, work on apps in the `expo/apps` folder.
+These will use the modules in the `expo/packages` folder, meaning you don't need to worry about accidentially erasing changes by working in `node_modules` 😬.
+
+### Working on modules
+
+For example if you want to work on `react-navigation`, clone it to `expo/packages/react-navigation` and run `yarn`, now projects like `expo/apps/native-component-list` will use your local copy of navigation and update when you make changes.
```

Comment:
For example, if you want to work on `react-navigation`, clone it to `expo/packages/react-navigation` and run `yarn`. Now projects like `expo/apps/native-component-list` will use your local copy of navigation and update when you make changes.